apps/mull-ui/src/app/components/event-card/event-card.tsx

```diff
@@ -26,33 +33,37 @@ export const EventCard = ({ event, style = {}, onClick, isJoined = false }: Even
   // TODO: Implement distance calculator
   const distance = 15;
 
+  const currentDate = new Date();
+  const isEventExpired = currentDate.getTime() > new Date(event.endDate).getTime();
   return (
     <div className="event-card-container button" onClick={onClick} style={style}>
       <img className="event-card-image" src={mediaUrl(event)} alt="Event" />
       <div className="event-card-datetime" data-testid="event-card-datetime">
         <div className="date-style">{`${day} ${month.toUpperCase()}`}</div>
         <div>{time.replace(/\s/g, '')}</div>
       </div>
-      {/* TODO: Add/remove user to event on press */}
-      <button
-        onClick={(e) => {
-          e.stopPropagation();
-          setJoined(!joined);
-          if (joined === false) {
-            joinEvent({ variables: { eventId: event.id } });
-          } else {
-            leaveEvent({ variables: { eventId: event.id } });
-          }
-        }}
-        className={`event-card-join ${joined ? 'joined' : 'not-joined'}`}
-        id={'event-card-join'}
-      >
-        {joined ? (
-          <FontAwesomeIcon icon={faCheck} />
-        ) : (
-          <img src="../../../assets/icons/join-icon.svg" className="join-icon" alt="Join" />
-        )}
-      </button>
+      {isEventExpired ? null : isEventOwner ? null : (
```

Comment:
💡 I've looked into it, tested on both javascript and jsx and I can't seem to find any issues with having a boolean operator in this ternary. Please try it out and let me know what the issue is. It looks a lot cleaner that way to me.